corehq/db.py

```diff
@@ -82,10 +82,14 @@ def dispose_all(self):
             self.dispose_engine(engine_id)
 
     def get_connection_string(self, engine_id):
-        return {
+        db_connection_map = {
             DEFAULT_ENGINE_ID: settings.SQL_REPORTING_DATABASE_URL,
             UCR_ENGINE_ID: settings.UCR_DATABASE_URL,
-        }.get(engine_id, settings.SQL_REPORTING_DATABASE_URL)
+        }
+
+        for custom_engine_id, custom_db_url in settings.CUSTOM_DATABASES:
+            db_connection_map[custom_engine_id] = custom_db_url
```

Comment:
Initially I liked the idea of being able to override the default connection map, but I can also see how that would be a really confusing configuration.